databases/week3/mealsharing.sql

```diff
@@ -0,0 +1,678 @@
+  )
+values
+  (
+    1,
+    'Dedang rendang',
+    "Its one of the best side dishes I tried in Malaysia",
+    1,
+    7,
+    "2022-03-03"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    2,
+    'Paratha',
+    'A freshly made Paratha which made us feel as we are back HOME, Rating it 5 cause it felt as dunked in oil',
+    2,
+    5,
+    "2022-02-13"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    3,
+    'Classic pizzetta',
+    'Home made italian & Mediterranean classic Pizza, it sound as it is described',
+    3,
+    8,
+    "2022-02-09"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    4,
+    'Shawarma',
+    'Love the smell and the presentation of the food, for sure will visit again',
+    4,
+    9,
+    "2022-03-03"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    5,
+    'Mezza Plates',
+    'Though the portion was comperatively smaller than expected it did taste super good',
+    5,
+    8,
+    "2022-03-03"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    6,
+    'XXX- Large Tallerken (8personer)',
+    'It was a great service at the restaurant, but the sorrounding was bit noisy',
+    6,
+    9,
+    "2022-02-01"
+  );
+insert into
+  review (
+    `id`,
+    `title`,
+    `description`,
+    `meal_id`,
+    `stars`,
+    `created_date`
+  )
+values
+  (
+    7,
+    'Rød grød med fløde',
+    "Det smager det samme som min mor laver",
+    7,
+    9,
+    "2022-03-03"
+  );
+SELECT
+  *
+FROM
+  review;
+--Get meals that has a price smaller than a specific price fx 90
+SELECT
+  *
+FROM
+  meal
+WHERE
+  price < 90;
+--Get meals that still has available reservations
+SELECT
+  *
+FROM
+  meal
+  JOIN reservation ON meal.id = reservation.meal_id
+WHERE
+  reservation.id = NULL;
+--Get meals that partially match a title. Rød grød med will match the meal with the title Rød grød med fløde
+SELECT
+  *
+FROM
+  meal
+WHERE
+  title LIKE 'Rød grød med%';
+--Get meals that has been created between two dates
+SELECT
+  *
+FROM
+  meal
+WHERE
+  created_date >= '2022-02-01'
+  AND created_date <= '2022-03-15';
+--Get only specific number of meals fx return only 5 meals
+SELECT
+  *
+FROM
+  meal
+LIMIT
+  2, 4;
+--Get the meals that have good reviews
+SELECT
+  *
+FROM
+  meal
+  JOIN review ON meal.id = review.meal_id
+WHERE
+  review.stars >= 7;
+--Get reservations for a specific meal sorted by created_date
+SELECT
+  *
+FROM
+  reservation
+  JOIN meal ON meal.id = reservation.meal_id
+WHERE
+  reservation.created_date = "2022-03-03";
```

Comment:
We want the reservations for a specific meal, not for a specific created date ;)